chassis/core/src/main/java/com/griddynamics/jagger/invoker/NonCircularExclusiveAccessLoadBalancer.java

```diff
@@ -0,0 +1,27 @@
+package com.griddynamics.jagger.invoker;
+
+import com.griddynamics.jagger.util.Pair;
+
+/**
+ * Subclass of {@link ExclusiveAccessLoadBalancer} that provides each pair of Q and E only once (does not circle an iteration).
+ * As a result it works as a finite load balancer over a predefined sequence of pairs.
```

Comment:
usually docs are not copy pasted from super class to subclass. I explicitly noted which class it extends in the doc  so it is easily to navigate for more details